demo/kubernetes/rook-operator.yaml

```diff
@@ -12,5 +12,9 @@ spec:
     spec:
       containers:
       - name: rook-operator
-        image: quay.io/rook/rook-operator:dev-2017-02-17-k8s
-        args: ["--container-version=dev-2017-02-17-k8s"]
+        image: quay.io/rook/rook-operator:dev-2017-02-23-k8s
```

Comment:
definitely agreed, we just have a bit of a chicken and egg. We have to publish the latest build that will support this. Right now the latest build is pre-kubernetes